reference.md

```diff
@@ -706,6 +720,66 @@ Field Name | Presence | Type | Description
       {
         "brand_id": "large_ride",
         "wait_time": 600
+      },
+      {
+        "brand_id": "shared_ride",
+        "wait_time": 450
+      }
```

Comment:
Same as above - this example should be updated to use `estimated_wait_time`

Reply:
updated